src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCode.java

```diff
@@ -8,6 +8,8 @@ public enum ErrorCode {
 
   CONCURRENCY_FAILURE("Unable to complete transaction due to concurrent transactions"),
 
+  DATASET_TOO_BIG("Response data set too big, add more filters"),
```

Comment:
Should this refer to sorting ("too big to be sorted"?), as it won't happen with regular Find command?

Reply:
Will change